coin-flip/index.html

```diff
@@ -1,17 +1,17 @@
 <!DOCTYPE html>
-<html>
+<html lang="en-US">
 <head>
 <title>Coin Flipper | My Digital Spot</title>
 <link rel="stylesheet" href="index.css">
 <script src="script.js"></script>
 </head>
-<center>
-<h1 class="header">Coin Flipper</h1>
-<div id="coin">
-<h1>Flip that coin!</h1>
-</div>
-<div id="coinimg"></div>
-<button onclick="getRandNumber(2)">Flip it again!</button>
-<div class="footer"><p>A simple coin flip app with clean design and nice colors. By Isaac Fiedler 2021. See https://github.com/chillzeus</p></div>
-</center>
+<body>
```

Comment:
You were missing your `<body>` tag before. I also indented your HTML inside that tag.